modelplugin/aether-1.0.0/yang/aether-subscriber@2020-08-18.yang

```diff
@@ -0,0 +1,134 @@
+module aether-subscriber {
+  namespace "http://opennetworking.org/aether/subscriber";
+  prefix sub;
+
+  import apn-profile{ prefix apn; }
+  import qos-profile{ prefix qos; }
+  import up-profile{ prefix up; }
+  import access-profile{ prefix ap; }
+
+  organization "Open Networking Foundation.";
+  contact "Scott Baker";
+  description "To generate JSON from this use command
+    pyang -f jtoxx test1.yang | python3 -m json.tool > test1.json
+    Copied from YangUIComponents project";
+
+  revision "2020-08-18" {
+    description "An Aether Subscriber";
+    reference "RFC 6087";
+  }
+
+  // User Equipment ID (UEID), Basically an IMSI range
+  // syntax:
+  //    123456789???????  ... wildcard at the end
+  //    123456???0123455  ... wildcard in the middle
+  //    1234567890000000-123456789000005 ... range
+  typedef ueid {
+      type string {
```

Comment:
maybe you had intended to make this more complex? Should have pattern like [0-9?]

Reply:
These were intended to be examples. I'll tweak the language in the next patch. Also should consider how to add a validator for it.